Makefile

```diff
@@ -363,25 +363,40 @@ release-build: bundle image-build bundle-build catalog-build ## Build operator,
 .PHONY:
 release-push: image-push bundle-push catalog-push ## Push operator, bundle + catalog images
 
+# It has to be the same namespace as ./config/default/kustomization.yaml -> namespace
+OPERATOR_NAMESPACE ?= backstage-system
+OLM_NAMESPACE ?= olm
+
 .PHONY: deploy-olm
 deploy-olm: ## Deploy the operator with OLM
-	kubectl apply -f config/samples/catalog-operator-group.yaml
-	sed "s/{{VERSION}}/$(subst /,\/,$(VERSION))/g" config/samples/catalog-subscription-template.yaml | sed "s/{{DEFAULT_OLM_NAMESPACE}}/$(subst /,\/,$(DEFAULT_OLM_NAMESPACE))/g" | kubectl apply -f -
+	kubectl -n ${OPERATOR_NAMESPACE} apply -f config/samples/catalog-operator-group.yaml
+	sed "s/{{VERSION}}/$(subst /,\/,$(VERSION))/g" config/samples/catalog-subscription-template.yaml | sed "s/{{OLM_NAMESPACE}}/$(subst /,\/,$(OLM_NAMESPACE))/g" | kubectl -n ${OPERATOR_NAMESPACE} apply -f -
 
 .PHONY: undeploy-olm
 undeploy-olm: ## Un-deploy the operator with OLM
-	-kubectl delete subscriptions.operators.coreos.com backstage-operator
-	-kubectl delete operatorgroup backstage-operator-group
-	-kubectl delete clusterserviceversion backstage-operator.$(VERSION)
+	-kubectl -n ${OPERATOR_NAMESPACE} delete subscriptions.operators.coreos.com backstage-operator
+	-kubectl -n ${OPERATOR_NAMESPACE} delete operatorgroup backstage-operator-group
+	-kubectl -n ${OPERATOR_NAMESPACE} delete clusterserviceversion backstage-operator.v$(VERSION)
 
-DEFAULT_OLM_NAMESPACE ?= openshift-marketplace
 .PHONY: catalog-update
 catalog-update: ## Update catalog source in the default namespace for catalogsource
-	-kubectl delete catalogsource backstage-operator -n $(DEFAULT_OLM_NAMESPACE)
-	sed "s/{{CATALOG_IMG}}/$(subst /,\/,$(CATALOG_IMG))/g" config/samples/catalog-source-template.yaml | kubectl apply -n $(DEFAULT_OLM_NAMESPACE) -f -
+	-kubectl delete catalogsource backstage-operator -n $(OLM_NAMESPACE)
+	sed "s/{{CATALOG_IMG}}/$(subst /,\/,$(CATALOG_IMG))/g" config/samples/catalog-source-template.yaml | kubectl apply -n $(OLM_NAMESPACE) -f -
 
+# Deploy on Openshift cluster using OLM (by default installed on Openshift)
 .PHONY: deploy-openshift
-deploy-openshift: release-build release-push catalog-update ## Deploy the operator on openshift cluster
+deploy-openshift: OLM_NAMESPACE=openshift-marketplace release-build release-push catalog-update deploy-olm ## Deploy the operator on openshift cluster
+
+.PHONY: install-olm
+install-olm: operator-sdk
```

Comment:
Well, both of them are quite trivial, I just considered 'install-olm' is like a quick start for using olm with operator, while undeploying operator is something one should to do with care in general case. 
My position is rather "no" then "yes", so if you have additional arguments, we can discuss it further :)

Reply:
No special argument, but my point is that, no matter how relatively "trivial" it can be, if we expose an `install-$something` target, I would expect to see the counterpart of it. Even when I'm going through a quick start/tutorial, it usually wraps up by cleaning all the resources that were created.
If you think it is trivial enough, in this case, I would suggest even not adding this `install-olm` task, to reduce the clutter in this Makefile and keep it simple.

Reply:
Not sure I understand your suggestion, as you can see `install-olm` is a part of "how to deploy on OLM" doc, which is the purpose of this task. 
IMO no matter how many targets we have on Makefile if they have purpose (even if redundant), are supported and documented.

Reply:
Okay, let's leave it as it is. We can always add it later if need be.